providers/class-two-factor-backup-codes.php

```diff
@@ -304,8 +304,8 @@ public function authentication_page( $user ) {
 	 * @return boolean
 	 */
 	public function validate_authentication( $user ) {
-		$backup_code = isset( $_POST['two-factor-backup-code'] ) ? sanitize_text_field( wp_unslash( $_POST['two-factor-backup-code'] ) ) : false;
-		return $this->validate_code( $user, filter_var( $backup_code, FILTER_SANITIZE_STRING ) );
+		$backup_code = isset( $_POST['two-factor-backup-code'] ) ? sanitize_text_field( wp_unslash( $_POST['two-factor-backup-code'] ) ) : '';
+		return $this->validate_code( $user, $backup_code );
```

Comment:
`$backup_code` is already sanitized with `sanitize_text_field()`; `filter_var()` is not really needed. To ensure that `$backup_code` is a string, I replaced `false` with an empty string.